src/main/java/org/broadinstitute/hellbender/engine/filters/MarkDuplicatesReadFilter.java

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.hellbender.engine.filters;
+
+import org.broadinstitute.hellbender.tools.spark.transforms.markduplicates.MarkDuplicatesSparkUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.io.Serializable;
+
+/**
+ * Filters out reads marked as duplicates.
+ */
+public final class MarkDuplicatesReadFilter extends ReadFilter implements Serializable {
```

Comment:
I'm unclear about the need for this class vs two separate filters, `OpticalDuplicateReadFilter` and `DuplicateReadFilter`.  Seems confusing.